README.md

```diff
@@ -700,6 +700,64 @@ Get builds that belong to this event
 model.getBuilds()
 ```
 
+### Template Model
+```js
+'use strict';
+const Model = require('screwdriver-models');
+const factory = Model.TemplateFactory.getInstance({
+    datastore
+});
+const config = {
+    name: 'testTemplate',
```

Comment:
But yeah I think St. John was envisioning some sort of `platform/step` sort of format